src/commands/moderation/rules.ts

```diff
@@ -55,7 +55,7 @@ const ROLE_EMBED = makeEmbed({
 export const rules: CommandDefinition = {
     name: 'rules',
     description: 'Sends the rules',
-    requiredPermissions: ['BanMembers'],
+    requirements: { permissions: ['BanMembers'] },
```

Comment:
Not a comment on the PR, but @benw202 : Does this make sense, should we take the opportunity to change this to ADMIN & MODERATOR roles?